nix/flake.nix

```diff
@@ -12,23 +12,27 @@
       };
     };
 
+    # NOTE: @2024-05 Nix flakes does not support getting git submodules of 'self'.
+    # refs:
+    # - https://discourse.nixos.org/t/get-nix-flake-to-include-git-submodule/30324
+    # - https://github.com/NixOS/nix/pull/7862
+    #
+    # ... In the meantime we kinda duplicate the dependencies here then replace the submodules with
+    # links to each repo in package sources.
     freetype2 = {
-      url = "github:wez/freetype2/de8b92dd7ec634e9e2b25ef534c54a3537555c11";
+      url = "github:wez/freetype2/VER-2-13-0";
       flake = false;
     };
-
     harfbuzz = {
-      url = "github:harfbuzz/harfbuzz/60841e26187576bff477c1a09ee2ffe544844abc";
+      url = "github:harfbuzz/harfbuzz/8.4.0";
       flake = false;
     };
-
     libpng = {
       url = "github:glennrp/libpng/8439534daa1d3a5705ba92e653eda9251246dd61";
```

Comment:
I didn't find a relevant tag for this commit, is that correct @wez 🤔

(I'm curious)
Is there something specific about that commit? or it was the last when you included that submodule and never needed to update?

Reply:
I have no specific recollection here. It's possible that updating to a newer rev would be a good idea.